bin/limitd

```diff
@@ -3,11 +3,40 @@
 process.title = 'limitd server';
 
 const config = require('../conf');
+const yaml = require('js-yaml');
+
+const agent = require('../lib/agent');
+const logger = agent.logger;
+const metrics = agent.metrics;
+const Configurator = require('../lib/configurator');
+
 const LimitdServer = require('../server');
 const server = new LimitdServer(config);
 
 server.start();
 
+server.on('started', () => {
+  if (config.configurator) {
+    config.configurator.metrics = metrics;
+    config.configurator.logger = logger;
+    config.configurator.currentVal = config.buckets;
+    const configurator = new Configurator(config.configurator);
+    configurator.on('changed', (data) => {
```

Comment:
I'd also listen for errors, even if it is not emitter right now it might get emitted on future and 💥

Reply:
<3